packages/augur-ui/src/modules/create-market/components/common.styles.less

```diff
@@ -1,38 +1,106 @@
-@import (reference) "~assets/styles/shared";
+@import (reference) '~assets/styles/shared';
 
 .Header {
-	.text-24-bold;
+  .text-24-bold;
 
-	color: @color-alternate-info-text;
+  color: @color-alternate-info-text;
 }
 
 .Subheaders {
-	display: flex;
-	flex-direction: column;
-			
-	> span:first-of-type {
-		.text-16-medium;
+  display: flex;
+  flex-direction: column;
 
-		color: @color-alternate-info-text;
-		margin-bottom: @size-8;
-	}
+  > span:first-of-type {
+    .text-16-medium;
 
-	> span:nth-of-type(2) {
-		.text-12;
+    color: @color-alternate-info-text;
+    margin-bottom: @size-8;
+  }
 
-		color: @color-tertiary-info-text;
-		line-height: @size-20;
-		margin-bottom: @size-8;
+  > span:nth-of-type(2) {
+    .text-12;
 
-		> a {
-			color: @color-primary-action;
-			margin-left: 3px;
-		}
+    color: @color-tertiary-info-text;
+    line-height: @size-20;
+    margin-bottom: @size-8;
+
+    > a {
+      color: @color-primary-action;
+      margin-left: 3px;
+    }
+  }
+
+  &:not(:first-of-type) {
+    > span:first-of-type {
+      margin-top: @size-32;
+    }
+  }
+}
+
+.NumberedList {
+  > li {
+    margin-bottom: @size-12;
+  }
+
+  > li:last-of-type {
+    margin-bottom: 0;
+
+    > button {
+			display: flex;
+			margin-left: 44px;
+
+      > svg {
+        height: @size-16;
+        width: @size-16;
+        margin-right: @size-8;
+
+        > path {
+          fill: @color-alternate-info-text;
+        }
+			}
+
+			&:hover:enabled {
+				> svg > path {
+					fill: @color-secondary-text;
+				}
+			}
+			
+			&:disabled {
+				> svg > path {
+					fill: @color-neutral;
+				}
+			}
+    }
+  }
+}
+
+.NumberedInput {
+  align-items: center;
+  display: flex;
+  flex-flow: row nowrap;
+
+  > span {
+    .text-14;
+
+    color: @color-alternate-info-text;
+    flex: 0 0 auto;
+    padding: 1rem;
```

Comment:
should this and the one below be @size-16?

Reply:
good catch, fixed